Tools/BlinkLocomotionTool/Scripts/BlinkVisuals.cs

```diff
@@ -78,6 +79,8 @@ private enum State
 	public bool validTarget { get; private set; }
 	public bool showValidTargetIndicator { private get; set; }
 
+	public Transform cameraRig { get; set; }
```

Comment:
Have BlinkTransformTool set a transform if needed or optionally, just provide the scale.